example/README.md

```diff
@@ -118,6 +118,37 @@ Run `terraform apply`.
 
 This will deploy your data persistence resources.
 
+## Configure and deploy the `db-migration-tf` root module
```

Comment:
I'm not sure how I feel about the DB migration being yet another "stack" to deploy. What is the reason for that?

I'm struggling to think of the case where it would be a problem to have `data-persistence` also handle the data migrations

Reply:
> That was the original reason that the resources in that stack were pulled into their own stack.

I interpreted our reasoning as just wanting to make sure that your data resources could not be affected by the state of the rest of your resources. To me, something that applies migrations to your database seems inherently connected to those resources, so having them bound up doesn't seem like an issue to me. I guess the concern of having migration resources with data storage resources is that you can't delete the migration resources in isolation. I'm not sure if that's a problem, but if we feel that it is, then okay. 

I also just feel like the user experience of `terraform apply` in three separate places starts to become not great, so I wish there was a way to keep this simpler